blog/2015/08/waivers-and-discounts.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: blog
+root: ../../..
+author: Raniere Silva
+title: "Fee Waivers and Discounted Fees"
+date: 2015-08-18
+time: "14:00:00"
+category: ["Steering Committee"]
+---
+<!-- start excerpt -->
+<p>Last week the Steering Committee met in person and one of the topic discussed was fee waivers and discounted fees for workshops and instructor training. Continue reading for our conclusions.</p>
+<!-- end excerpt -->
+<p>In the past, Software Carpentry has granted fee waivers and discounted fees in several cases when our staff was involved in finding instructors for a workshop. For example, Software Carpentry has waived the fees for the Workshops for Women in Science and Engineering (WISE) because these events contribute to closing the gender gap in computational science. The SCF has also waived fees for workshops run in countries with weak economies and relatively lower expenditures on research and development.</p>
+<p>Software Carpentry will give grant waivers and discounted fees to enable workshops</p>
+<ul>
+<li>which aim to help any disadvantaged groups improve their computational lab skills, as Software Carpentry believes that this in turn helps them progress with their careers and increases their social mobility,</li>
+<li>in countries with weak economies (gauged based on GDP per capita, average and minimum salary, unemployment rate etc.) and relatively low investment in research and development, and</li>
+<li>in cases not mentioned above at the discretion of the Software Carpentry Steering Committee.</li>
+</ul>
+<p>To help ensure that any workshop given a waiver or a discount was accessible to the maximum number of learners, the Foundation requires that the host presents documented information about the total costs and total income (from entry fees, if they charge them) from running the workshop. This implies that hosts may charge an entry fee of participants while still getting a discount or a waiver, as long as entry fees go directly to fund the costs of the workshop. These can include, but are not limited to:</p>
+<ul>
+<li>transportation and accommodations for instructors,</li>
+<li>coffee/tea/juice, snacks, lunch for instructors and learners during the workshop,</li>
+<li>stickies, marker pens, ...</li>
+<li>refunds of the fee to learners that attend the workshop (also known as a no-show fee)</li>
+</ul>
+<p>And for the future, we hope for us be capable to offer financial help for a few workshops and instructors training each year.</p>
```

Comment:
As a long term goal,  the foundation hopes to be able to give financial help to a few workshops and instructors trainings each year.